tests/js/common/aql/aql-search-optimize-top-k.js

```diff
@@ -44,18 +50,18 @@ function testOptimizeTopK() {
       }
 
       db._createView("v_search_tfidf", "arangosearch", {
-        links: {"c": {fields: {"f": {}}}},
+        links: { "c": { fields: { "f": {}, "a": {}, "x": {} } } },
         optimizeTopK: ["TFIDF(@doc) DESC"]
       });
 
-      db.c.ensureIndex({name: "i_bm25", type: "inverted", fields: ["f"], optimizeTopK: ["BM25(@doc) DESC"]});
-      db._createView("v_alias_bm25", "search-alias", {indexes: [{collection: "c", index: "i_bm25"}]});
+      db.c.ensureIndex({ name: "i_bm25", type: "inverted", fields: ["f", "a", "x"], optimizeTopK: ["BM25(@doc) DESC"] });
```

Comment:
Data has fields  f, **c**, x but index indexes f, **a**, x - is that intentional?

Reply:
Yes

Reply:
But I will change it